### PR TITLE
Add gallery to Info block

### PR DESCRIPTION
**Pull request by ivan-berezhnov**

### CODE REVIEW

File: docroot/modules/custom/twelve_app/js/seven_summits/components/MountainInfoModal.vue [hunk 1]
```diff
@@ -46,10 +50,22 @@
   import Modal from '../../components/Modal';
   import { mapMutations, mapState, mapActions } from 'vuex';
   import MountainMixin from "../mixins/Mountain";
+  // Documentation for gallery: https://github.com/ChristophAnastasiades/Lingallery
+  import Lingallery from 'lingallery';
```
Comment (anpolimus) [inline docroot/modules/custom/twelve_app/js/seven_summits/components/MountainInfoModal.vue:54]: There is no need to import this component if it is already imported in a child

**Comment by retif** [comment]
Image style configuration is missing

File: docroot/modules/custom/twelve_app/js/seven_summits/components/MountainInfoModal.vue (hunk 1 above)
Comment (d3-ivan-berezhnov) [reply docroot/modules/custom/twelve_app/js/seven_summits/components/MountainInfoModal.vue]: Fixed

**Comment by d3-ivan-berezhnov** [comment]
> Image style configuration is missing for 'mountain_thumbnail'

fixed

File: docroot/modules/custom/twelve_app/js/seven_summits/components/MountainInfoModal.vue (hunk 1 above)
Comment (ivan-berezhnov) [reply docroot/modules/custom/twelve_app/js/seven_summits/components/MountainInfoModal.vue]: Fixed

**Comment by ivan-berezhnov** [comment]
> Image style configuration is missing for 'mountain_thumbnail'

Fixed

**Pull request merged**